### PR TITLE
Setting a name for the otherwise blank transient key

### PR DESCRIPTION
**Pull request by bcampeau**

### CODE REVIEW

File: lib/class-sp-sync-meta.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ class SP_Sync_Meta {
 	 * @access protected
 	 * @var string
```
Comment (tomharrigan) [inline lib/class-sp-sync-meta.php]: Should this be a mixed string/boolean based on what the set_transient on line 189 is doing?

Comment (bcampeau) [inline lib/class-sp-sync-meta.php]: No, the key itself is always a string.

**Comment by tomharrigan** [comment]
:hamster:

File: lib/class-sp-sync-meta.php (hunk 1 above)
Comment (tomharrigan) [inline lib/class-sp-sync-meta.php]: silly me, sorry

**Pull request merged**